lib/stat_tracker.rb

```diff
@@ -1,21 +1,29 @@
 class StatTracker
-  attr_reader :games_path, :teams_path, :game_teams_path
+  attr_reader :games_path, :teams_path, :game_teams_path, :game_collection, :team_collection
+
   def initialize(stat_tracker_params)
     @games_path = stat_tracker_params[:games]
     @teams_path = stat_tracker_params[:teams]
     @game_teams_path = stat_tracker_params[:game_teams]
+    @game_collection = GameCollection.new(@games_path)
```

Comment:
I moved these up here so that we could call the actual `Collection` separately from `Collection.all` in the methods below.

Reply:
Agreed. Far easier this way.